### PR TITLE
Fixed inr type

### PR DESCRIPTION
**Pull request by aniket-teltumbade-au9**

### CODE REVIEW

**Comment by s7dhansh** [comment]
Task linked: [CU-2v7g1n4 React Hook Form MUi package input type inr ffeild fixing](https://app.clickup.com/t/2v7g1n4)

File: react-hook-form-mui/src/forms/formikToMuiProps.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import {getIn} from 'formik';
```
Comment (s7dhansh) [inline react-hook-form-mui/src/forms/formikToMuiProps.js:1]: why do you need formikToMuiProps in react hook form?

Comment (s7dhansh) [reply react-hook-form-mui/src/forms/formikToMuiProps.js:1]: @aniket-teltumbade-au9 @coolamit1240

**Comment by aniket-teltumbade-au9** [comment]
Creating a new PR

File: react-hook-form-mui/src/forms/formikToMuiProps.js (hunk 1 above)
Comment (aniket-teltumbade-au9) [reply react-hook-form-mui/src/forms/formikToMuiProps.js:1]: Sir, It was called in /src/forms/Radio.js,  but it wasn't declared.

Comment (aniket-teltumbade-au9) [reply react-hook-form-mui/src/forms/formikToMuiProps.js]: @s7dhansh  Sir I have pushed new changes. Please review it.

**Comment by aniket-teltumbade-au9** [comment]
https://github.com/settlin/node-monorepo/pull/111

**Pull request closed without merging**